core/src/test/resources/test_assets.json

```diff
@@ -14,7 +14,11 @@
       "withdraw": {
         "enabled": true,
         "min_amount": 1,
```

Comment:
This would require use to remove the validation from SEP-24 implementation. I would prefer to keep the scope of this PR specific to SEP-6. I can create a Jira to track it though.

Reply:
ANCHOR-381